Binary files heimdal-0.7-old/cf/.find-func-no-libs2.m4.swp and heimdal-0.7/cf/.find-func-no-libs2.m4.swp differ
Index: heimdal/cf/db.m4
===================================================================
--- heimdal.orig/cf/db.m4
+++ heimdal/cf/db.m4
@@ -18,24 +18,13 @@ db_type=unknown
 
 if test "$enable_berkeley_db" != no; then
 
-  AC_CHECK_HEADERS([				\
-	db4/db.h				\
-	db3/db.h				\
-	db.h					\
-	db_185.h				\
-  ])
+  AC_CHECK_HEADERS([db.h])
 
 dnl db_create is used by db3 and db4
 
-  AC_FIND_FUNC_NO_LIBS(db_create, db4 db3 db, [
+  AC_FIND_FUNC_NO_LIBS(db_create, db, [
   #include <stdio.h>
-  #ifdef HAVE_DB4_DB_H
-  #include <db4/db.h>
-  #elif defined(HAVE_DB3_DB_H)
-  #include <db3/db.h>
-  #else
   #include <db.h>
-  #endif
   ],[NULL, NULL, 0])
 
   if test "$ac_cv_func_db_create" = "yes"; then
@@ -50,17 +39,9 @@ dnl db_create is used by db3 and db4
 
 dnl dbopen is used by db1/db2
 
-    AC_FIND_FUNC_NO_LIBS(dbopen, db2 db, [
+    AC_FIND_FUNC_NO_LIBS(dbopen, db, [
     #include <stdio.h>
-    #if defined(HAVE_DB2_DB_H)
-    #include <db2/db.h>
-    #elif defined(HAVE_DB_185_H)
-    #include <db_185.h>
-    #elif defined(HAVE_DB_H)
     #include <db.h>
-    #else
-    #error no db.h
-    #endif
     ],[NULL, 0, 0, 0, NULL])
 
     if test "$ac_cv_func_dbopen" = "yes"; then
Index: heimdal/lib/hdb/db.c
===================================================================
--- heimdal.orig/lib/hdb/db.c
+++ heimdal/lib/hdb/db.c
@@ -37,11 +37,7 @@ RCSID("$Id: db.c 20215 2007-02-09 21:59:
 
 #if HAVE_DB1
 
-#if defined(HAVE_DB_185_H)
-#include <db_185.h>
-#elif defined(HAVE_DB_H)
 #include <db.h>
-#endif
 
 static krb5_error_code
 DB_close(krb5_context context, HDB *db)
Index: heimdal/lib/hdb/db3.c
===================================================================
--- heimdal.orig/lib/hdb/db3.c
+++ heimdal/lib/hdb/db3.c
@@ -37,13 +37,7 @@ RCSID("$Id: db3.c 21610 2007-07-17 07:10
 
 #if HAVE_DB3
 
-#ifdef HAVE_DB4_DB_H
-#include <db4/db.h>
-#elif defined(HAVE_DB3_DB_H)
-#include <db3/db.h>
-#else
 #include <db.h>
-#endif
 
 static krb5_error_code
 DB_close(krb5_context context, HDB *db)
Index: heimdal/lib/roken/getcap.c
===================================================================
--- heimdal.orig/lib/roken/getcap.c
+++ heimdal/lib/roken/getcap.c
@@ -38,11 +38,13 @@
 #include "roken.h"
 RCSID("$Id: getcap.c 16561 2006-01-13 14:25:32Z lha $");
 
+#if defined(HAVE_DBOPEN) && defined(HAVE_DB_H)
+#define USE_DB
+#endif
+
 #include <sys/types.h>
 #include <ctype.h>
-#if defined(HAVE_DB_185_H)
-#include <db_185.h>
-#elif defined(HAVE_DB_H)
+#ifdef USE_DB
 #include <db.h>
 #endif
 #include <errno.h>	
Index: heimdal/lib/roken/ndbm_wrap.c
===================================================================
--- heimdal.orig/lib/roken/ndbm_wrap.c
+++ heimdal/lib/roken/ndbm_wrap.c
@@ -37,13 +37,7 @@ RCSID("$Id: ndbm_wrap.c 21634 2007-07-17
 #endif
 
 #include "ndbm_wrap.h"
-#if defined(HAVE_DB4_DB_H)
-#include <db4/db.h>
-#elif defined(HAVE_DB3_DB_H)
-#include <db3/db.h>
-#else
 #include <db.h>
-#endif
 
 #include <stdio.h>
 #include <stdlib.h>
